fearless/Modules/AccountManagement/AccountManagementViewController.swift

```diff
@@ -78,10 +76,8 @@ final class AccountManagementViewController: UIViewController {
         tableView.contentInset = .init(top: 0, left: 0, bottom: bottomInset, right: 0)
 
         tableView.register(R.nib.accountTableViewCell)
-        tableView.register(
-            UINib(resource: R.nib.iconTitleHeaderView),
-            forHeaderFooterViewReuseIdentifier: Constants.headerId
-        )
+
+        tableView.rowHeight = Constants.cellHeight
```

Comment:
Why do we need to set row height explicitly?

Reply:
Those cells doesn't support automatic height.